src/http-client/fetch-http-client.js

```diff
@@ -0,0 +1,89 @@
+
+const { version } = require('../version');
+
+
+class FetchHttpClient {
+    constructor(options) {
+        require('./setup-fetch');
```

Comment:
Not sure, but we should probably call this function only once before `FetchHttpClient` declaration. It would be better if `setup-fetch` module would export such functions as `fetch`, `Headers` etc. So then , we dont have to check if `fetch` and `Headers` exist, `setup-fetch` module will make it for us and return required functionality.

```
import {fetch, Headers, ...} from './setup-fetch';
```

Reply:
You are probably right, I will look how we can use it as a ponyfill.

Reply:
Why not just use cross-platform axios?

Reply:
@tjifyodor shooting birds with tank, probably. Just no need in `axios`